Snowflake.Data.Tests/SFBindTestIT.cs

```diff
@@ -686,6 +687,83 @@ public void testPutArrayBind()
                 conn.Close();
             }
         }
+        
+        [Test]
+        public void testPutArrayBindWorkDespiteOfProblematicTypeNameHandlingValue()
+        {
+            JsonConvert.DefaultSettings = () => new JsonSerializerSettings {
+                TypeNameHandling = TypeNameHandling.Auto
+            };
+            
+            
+            using (IDbConnection conn = new SnowflakeDbConnection())
+            {
+                conn.ConnectionString = ConnectionString;
+                conn.Open();
+
+                using (IDbCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = "create or replace table testPutArrayBind(cola REAL, colb TEXT, colc NUMBER(38,0))";
+                    int count = cmd.ExecuteNonQuery();
+                    Assert.AreEqual(0, count);
+
+                    string insertCommand = "insert into testPutArrayBind values (?, ?, ?)";
+                    cmd.CommandText = insertCommand;
+
+                    int total = 250;
+                    
+                    List<double> arrdouble = new List<double>();
+                    for (int i=0; i<total; i++)
+                    {
+                        arrdouble.Add(i * 10 + 1);
+                        arrdouble.Add(i * 10 + 2);
+                        arrdouble.Add(i * 10 + 3);
+                    }
+                    var p1 = cmd.CreateParameter();
+                    p1.ParameterName = "1";
+                    p1.DbType = DbType.Double;
+                    p1.Value = arrdouble.ToArray();
+                    cmd.Parameters.Add(p1);
+                    
+                    List<string> arrstring = new List<string>();
+                    for (int i=0; i<total; i++)
+                    {
+                        arrstring.Add("str1");

```

Comment:
don't we want to add some iterative suffix as for double and int tests?

Reply:
ok